tests/drawer/test_tape_mpl.py

```diff
@@ -251,6 +251,33 @@ def test_CNOT(self):
         assert len(ax.lines) == 5
         plt.close()
 
+    def test_ControlledQubitUnitary(self):
+        """Test ControlledQubitUnitary gets a special call."""
+
+        with QuantumTape() as tape:
+            qml.ControlledQubitUnitary(
+                qml.matrix(qml.RX)(0, 0),
+                control_wires=[0, 1, 2, 3],
+                wires=[4],
+                control_values="1010",
+            )
+
+        _, ax = tape_mpl(tape)
+        layer = 0
+
+        assert len(ax.patches) == 5
+        assert ax.patches[0].center == (layer, 0)
```

Comment:
While this is *almost* an excellent test, it only checks whether a control circle exists and doesn't check the type of control circle.  

We should check that the facecolor of control-on-zero circles matches the colour of the background and that the facecolor of the control-on-one circles matches the colour of the lines.

You can see how to test such a thing in [`tests/drawer/test_mpldrawer.py`](https://github.com/PennyLaneAI/pennylane/blob/d539835b11457ae621cd4fab04f3d20184eeec5e/tests/drawer/test_mpldrawer.py#L334). Relevant tests to look at are `test_ctrl_on_zero`, `test_ctrl_circ`, and `test_ctrlo_circ`.

For open circles:
```
        assert circ.get_facecolor() == to_rgba(plt.rcParams["axes.facecolor"])
```
and for closed circles:
```
        assert circ.get_facecolor() == to_rgba(plt.rcParams["lines.color"])
```

Reply:
Even though we have now removed the special case for `qml.ControlledQubitUnitary`, I am keeping this test with your modifications to verify the changes work as expected. Mabye it needs to be moved from `TestSpecialGates` to somewhere else?

Reply:
There's a `TestControlledGates` class below.  I think it would make sense to move this test there. And we could instead call it `test_control_values` since it will instead be testing general control-values behaviour instead of specific `ControlledQubitUnitary` behaviour.

Reply:
Sounds good, done!